### PR TITLE
Fix for Issue #16: Variable interpolation failure

### PR DESCRIPTION
**Pull request by prateekcr**
Fix for "https://github.com/secure-device-onboard/iot-platform-sdk/issues/16"

Updating to use Apache Commons configuration to load properties, instead
of using Java's native Properties and System classes.

Signed-off-by: Chandrakar, Prateek <prateek.chandrakar@intel.com>

### CODE REVIEW

File: ocs/fsimpl/pom.xml [hunk 1]
```diff
@@ -54,6 +54,16 @@
       <artifactId>reactor-core</artifactId>
     </dependency>
 
+    <dependency>
```
Comment (trbehera) [inline ocs/fsimpl/pom.xml:57]: Please update the commit subject to describe the changes within the commit.

The link to issue can be put within the commit description.

File: ocs/fsimpl/src/main/java/org/sdo/iotplatformsdk/ocs/fsimpl/fs/FsPropertiesLoader.java [hunk 2]
```diff
@@ -3,10 +3,14 @@
 
 package org.sdo.iotplatformsdk.ocs.fsimpl.fs;
 
-import java.io.FileReader;
-import java.io.IOException;
-import java.util.Properties;
 import java.util.concurrent.ConcurrentHashMap;
+import org.apache.commons.configuration2.EnvironmentConfiguration;
```
Comment (trbehera) [inline ocs/fsimpl/src/main/java/org/sdo/iotplatformsdk/ocs/fsimpl/fs/FsPropertiesLoader.java:7]: Is there a possibility to create a common class for reading the properties and then extending that class in each of the services?

File: ocs/fsimpl/pom.xml (hunk 1 above)
Comment (prateekcr) [reply ocs/fsimpl/pom.xml:57]: Done.

File: ocs/fsimpl/src/main/java/org/sdo/iotplatformsdk/ocs/fsimpl/fs/FsPropertiesLoader.java (hunk 2 above)
Comment (prateekcr) [reply ocs/fsimpl/src/main/java/org/sdo/iotplatformsdk/ocs/fsimpl/fs/FsPropertiesLoader.java:7]: The 3 components OPS, OCS and To0Scheduler are tied by beans defined in common/rest module as per the REST contract. So the common class could go there, and the properties loader classes would extend/implement it.
However, it is not mandatory for each component  to handle properties in the same manner, and it is not a part of any REST contract between the components, which is why these are separated out for each component, incidentally having the same class-level contents in them.

**Pull request merged**